app/src/main/java/app/gamenative/ui/screen/library/LibraryAppScreen.kt

```
@@ -452,13 +456,49 @@ private fun formatBytes(bytes: Long): String {
     val mb = kb * 1024
     val gb = mb * 1024
     return when {
-        bytes >= gb -> String.format("%.1f GB", bytes / gb)
-        bytes >= mb -> String.format("%.1f MB", bytes / mb)
-        bytes >= kb -> String.format("%.1f KB", bytes / kb)
+        bytes >= gb -> String.format("%.2f GB", bytes / gb)
```

detekt (1.23.8)

[warning] String.format("%.2f GB", bytes / gb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)